docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 1 (3 - 5 nodes)       | s      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 2 (3 - 7 nodes)       | m      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 6 (3 - 15 nodes)      | l      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 10 (3 - 23 nodes)     | xl     |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 1 (1 - 3 nodes)       | xs     |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 1 (3 - 5 nodes)       | s      |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 2 (3 - 7 nodes)       | m      |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 3 (3 - 9 nodes)       | l      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 1 (1 - 1 instances)   | xs     |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 3 (1 - 3 instances)   | s      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 6 (1 - 6 instances)   | m      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 18 (1 - 18 instances) | l      |
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+
+The output lists the CrateDB products that are available in your default region.
+The following node sizes (tiers) are available for the deployment of CrateDB
+products: **xs, s, m, l, xl**.
+
+Pick the node size ``xs`` for your test purpose.
+
+It is the smallest node size for a CrateDB product. According to the list above,
+this is the CrateDB cluster ``cratedb.az-gp1``.
+
+
+.. _launch-cluster:
+
+Launch the cluster
+==================
+
+Now that you have created an organization and project, you can deploy the
+cluster by running this command:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1
+
+.. TIP::
+
+  For an explaining list of the required arguments in this command check out the
+  `Croud CLI Reference`_.
+
+If the operation was successful, you should get the deployment details returned:
```

Comment:
like before, work this command output into the previous example block

explain it afterward